providers/base/units/bluetooth/manifest.pxu

```diff
@@ -7,3 +7,8 @@ unit: manifest entry
 id: has_bt_smart
 _name: A Bluetooth Module with Smart (4.0 or later) Support
 value-type: bool
+
+unit: manifest entry
+id: has_bt_obex_support
+_name: A Bluetooth Module with OBEX Support
```

Comment:
This is not the problem we are trying to solve. All the BT modules have OBEX support for what I know.

The problem is that we don't have the infrastructure to support testing OBEX everywhere in the lab. In order for the BT OBEX test to work, you need an OBEX server configured and close enough to the DUT to send and receive files to.

The manifest should be named something like `bt_obex_server_available` instead, but then we have to make it very clear that this is required for QA or ODM to run, otherwise they may just set this manifest to `False` and the related test(s) would be skipped...

Reply:
The fact that we are going to abuse this manifest key for the purpose of skipping the test in SRU in my opinion doesn't make it less valuable. In the form I have created it, it is a question about a capability. We may decide to lie about the capability in our own testing for our own purposes, but if we create `bt_obex_server_available`, that requires the description to be updated to say to always set it to true. While if we call it what it actually is, I think it is more clear to everybody when to set it true (which is basically all the times)